router/h2/src/test/scala/com/twitter/finagle/buoyant/h2/H2FailureAccrualFactoryTest.scala

```diff
@@ -0,0 +1,107 @@
+package com.twitter.finagle.buoyant.h2
+
+import com.twitter.concurrent.AsyncQueue
+import com.twitter.finagle.buoyant.h2.service.{H2Classifier, H2ReqRep, H2ReqRepFrame}
+import com.twitter.finagle.context.Contexts
+import com.twitter.finagle.liveness.FailureAccrualPolicy
+import com.twitter.finagle.service.ResponseClass
+import com.twitter.finagle.stats.InMemoryStatsReceiver
+import com.twitter.finagle.{FactoryToService, Service, ServiceFactory, Status => FStatus}
+import com.twitter.util.{Duration, Future, MockTimer, Return}
+import io.buoyant.router.context.h2.H2ClassifierCtx
+import io.buoyant.test.FunSuite
+import java.util
+import scala.{Stream => SStream}
+
+class H2FailureAccrualFactoryTest extends FunSuite {
```

Comment:
Nope.  The FAF tests are quite a bit more extensive but obviously don't cover the stream classifier aspect.